expreduce/graphics/graphics.go

```diff
@@ -179,12 +179,6 @@ func Render(expr expreduceapi.Ex) (chart.Chart, error) {
 	graph := chart.Chart{
 		Width:  360,
 		Height: 220,
-		XAxis: chart.XAxis{
-			Style: chart.Style{Show: true},
```

Comment:
The `incompatible` API required `Show` to be set, but the 2.0 released version instead has a `Hidden` boolean so that elements are default enabled.  We forked from this 2.0 API with the `Hidden` field.